README.md

```diff
@@ -61,32 +84,34 @@ For more details, see our [quickstart guide](https://docs.odigos.io/intro).
 
 ### Managed
 
-|                         | Traces  | Metrics | Logs |
-|-------------------------| ------- | ------- |------|
-| New Relic               | ✅      | ✅      | ✅    |
-| Datadog                 | ✅      | ✅      | ✅    |
-| Grafana Cloud           | ✅      | ✅      | ✅    |
-| Honeycomb               | ✅      | ✅      | ✅    |
-| Chronosphere            | ✅      | ✅      |       |
-| Logz.io                 | ✅      | ✅      | ✅    |
-| qryn.cloud              | ✅      | ✅      | ✅    |
-| OpsVerse                | ✅      | ✅      | ✅    |
-| Dynatrace               | ✅      | ✅      | ✅    |
-| AWS S3                  | ✅      | ✅      | ✅    |
-| Google Cloud Monitoring | ✅      |         | ✅    |
-| Google Cloud Storage    | ✅      |         | ✅    |
-| Azure Blob Storage      | ✅      |         | ✅    |
-| Splunk                  | ✅      |         |      |
-| Lightstep               | ✅      |         |      |
-| Sentry                  | ✅      |         |      |
-| Axiom                   | ✅      |         | ✅   |
-| Sumo Logic              | ✅      | ✅      | ✅   |
-| Coralogix               | ✅      | ✅      | ✅   |
+
+|                         | Traces | Metrics | Logs |
+| ------------------------- | -------- | --------- | ------ |
+| New Relic               | ✅     | ✅      | ✅   |
+| Datadog                 | ✅     | ✅      | ✅   |
+| Grafana Cloud           | ✅     | ✅      | ✅   |
+| Honeycomb               | ✅     | ✅      | ✅   |
+| Chronosphere            | ✅     | ✅      |      |
+| Logz.io                 | ✅     | ✅      | ✅   |
+| qryn.cloud              | ✅     | ✅      | ✅   |
+| OpsVerse                | ✅     | ✅      | ✅   |
+| Dynatrace               | ✅     | ✅      | ✅   |
+| AWS S3                  | ✅     | ✅      | ✅   |
+| Google Cloud Monitoring | ✅     |         | ✅   |
+| Google Cloud Storage    | ✅     |         | ✅   |
+| Azure Blob Storage      | ✅     |         | ✅   |
+| Splunk                  | ✅     |         |      |
+| Lightstep               | ✅     |         |      |
+| Sentry                  | ✅     |         |      |
+| Axiom                   | ✅     |         | ✅   |
+| Sumo Logic              | ✅     | ✅      | ✅   |
+| Coralogix               | ✅     | ✅      | ✅   |
 
 ### Open Source
 
+
 |               | Traces | Metrics | Logs |
-| ------------- | ------ | ------- | ---- |
+| --------------- | -------- | --------- | ------ |
```

Comment:
Also this list is not up-to-date.

For example, it is missing ClickHouse and otlp (gRPC, http) destinations